rero/rero-ils-ui
# contribution: implements organisation agents

The shared folder allows you to share code between several projects.
The project tests have been adapted with the new structure of the Shared module.
Displays organisation agent data in the document brief and detailed view.
Allows the librarian to search for organisation agent authority record in the MEF server when cataloguing.

* Fixes patron history with status "In transit to house".
* Closes rero/rero-ils#1360

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Depedencies

- https://github.com/rero/rero-ils/pull/1274

## Why are you opening this PR?

- Contributions: Person and Organisation

## How to test?

- Check views person or organisation on public and admin.
- Check Typeahead on document for person and organisation.

## Code review check list
